lib/new_relic/agent/instrumentation/action_controller_other_subscriber.rb

```diff
@@ -28,7 +28,7 @@ def metric_name(name, payload)
 
         def controller_name_for_metric(payload)
           # redirect_to
-          return payload[:request].controller_class.controller_path if payload[:request] && payload[:request].controller_class
+          return payload[:request].controller_class.controller_path if payload[:request]&.controller_class
```

Comment:
Unfortunately this line merge conflicts with the PR I merged this morning, i made some changes to this method.

Reply:
I thought that might happen. Fixed in: 054c25bb827ddd3a55e560bf843074e2c1553fc1